gui/treemodel.cc

```diff
@@ -161,35 +161,13 @@ void Model::loadData(std::unique_ptr<Item> data)
     endResetModel();
 }
 
-void Model::updateCells(Context *ctx)
+void Model::updateElements(Context *ctx, std::vector<IdString> elements)
 {
     if (!ctx)
```

Comment:
Unrelated: I might never get used to having pointers instead of references here… I don't see the point.

Reply:
++